pom.xml

```diff
@@ -0,0 +1,301 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.xebia</groupId>
+    <artifactId>headerbuddy</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
```

Comment:
Please make sure that your pom versioning is in line with your git versioning. This can be automated.